src/sentry/search/events/builder.py

```diff
@@ -380,3 +380,54 @@ def resolve_top_event_conditions(
         else:
             final_condition = None
         return final_condition
+
+
+class HistogramQueryBuilder(QueryBuilder):
+    def __init__(
+        self,
+        num_buckets: int,
+        histogram_column: str,
+        histogram_rows: Optional[int],
+        histogram_params: HistogramParams,
+        key_column: Optional[str],
+        field_names: Optional[List[Union[str, Any, None]]],
+        groupby: Optional[List[str]],
+        *args: Any,
+        **kwargs: Any,
+    ):
+        super().__init__(*args, **kwargs)
+        self.additional_groupby = groupby
+        selected_columns = kwargs["selected_columns"]
+        # Reset&Ignore the columns from the QueryBuilder
+        self.aggregates: List[CurriedFunction] = []
+        self.columns = [self.resolve_column("count()")]
```

Comment:
Mmm I prefer the append style, since the resolved_Histogram column needs to be referenced in the conditions + orderby

Though I could move L409 before this line to avoid having to append which is more readable :+1: